core/src/main/kotlin/com/tschuchort/compiletesting/KotlinCompilation.kt

```diff
@@ -678,3 +681,15 @@ class KotlinCompilation : AbstractKotlinCompilation<K2JVMCompilerArguments>() {
 		const val OPTION_KAPT_KOTLIN_GENERATED = "kapt.kotlin.generated"
     }
 }
+
+/**
+ * Adds the output directory of [previousResult] to the classpath of this compilation. This is a convenience for
+ * ```
+ * this.classpaths += previousResult.outputDirectory
+ * ```
+ */
+fun KotlinCompilation.addPreviousResultToClasspath(
+	previousResult: KotlinCompilation.Result
+): KotlinCompilation = apply {
+	classpaths += previousResult.outputDirectory
+}
```

Comment:
I'm unsure about this. I personally find it convenient, but it doesn't need to be part of this library itself. I'm okay to drop it (I made it a separate commit).

Reply:
Since it's an extension function, I don't see why not.